### PR TITLE
[typo](docs)Fix the jump link 404 in release-verify

### PR DESCRIPTION
**Pull request by wangtianyi2004**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/community/release-and-verify/release-verify.md [hunk 1]
```diff
@@ -97,6 +97,6 @@ INFO Totally checked 5611 files, valid: 3926, invalid: 0, ignored: 1685, fixed:
 
 请参阅各组件的编译文档验证编译。
 
-* Doris 主代码编译，请参阅 [编译文档](../../docs/install/source-install/compilation)
-* Flink Doris Connector 编译，请参阅 [编译文档](../../docs/ecosystem/flink-doris-connector)
-* Spark Doris Connector 编译，请参阅 [编译文档](../../docs/ecosystem/spark-doris-connector)
+* Doris 主代码编译，请参阅 [编译文档](../../docs/install/source-install/compilation.md)
```
Comment (hf200012) [inline docs/zh-CN/community/release-and-verify/release-verify.md:100]:
```suggestion
* Doris 主代码编译，请参阅 [编译文档](/docs/install/source-install/compilation)
```

File: docs/zh-CN/community/release-and-verify/release-verify.md [hunk 2]
```diff
@@ -97,6 +97,6 @@ INFO Totally checked 5611 files, valid: 3926, invalid: 0, ignored: 1685, fixed:
...
+* Flink Doris Connector 编译，请参阅 [编译文档](../../docs/ecosystem/flink-doris-connector.md)
```
Comment (hf200012) [inline docs/zh-CN/community/release-and-verify/release-verify.md:101]:
```suggestion
* Flink Doris Connector 编译，请参阅 [编译文档](/docs/ecosystem/flink-doris-connector)
```

File: docs/zh-CN/community/release-and-verify/release-verify.md [hunk 3]
```diff
@@ -97,6 +97,6 @@ INFO Totally checked 5611 files, valid: 3926, invalid: 0, ignored: 1685, fixed:
...
+* Spark Doris Connector 编译，请参阅 [编译文档](../../docs/ecosystem/spark-doris-connector.md)
```
Comment (hf200012) [inline docs/zh-CN/community/release-and-verify/release-verify.md:102]:
```suggestion
* Spark Doris Connector 编译，请参阅 [编译文档](/docs/ecosystem/spark-doris-connector)
```

File: docs/zh-CN/community/how-to-contribute/how-to-contribute.md [hunk 4]
```diff
@@ -78,7 +78,7 @@ under the License.
 
 ## 修改代码和提交PR（Pull Request）
 
-您可以下载代码，编译安装，部署运行试一试（可以参考[编译文档](/docs/install/source-install/compilation.md)，看看是否与您预想的一样工作。如果有问题，您可以直接联系我们，提 Issue 或者通过阅读和分析源代码自己修复。
+您可以下载代码，编译安装，部署运行试一试（可以参考[编译文档](../../../zh-CN/docs/install/source-install/compilation.md)，看看是否与您预想的一样工作。如果有问题，您可以直接联系我们，提 Issue 或者通过阅读和分析源代码自己修复。
```
Comment (hf200012) [inline docs/zh-CN/community/how-to-contribute/how-to-contribute.md:81]:
```suggestion
您可以下载代码，编译安装，部署运行试一试（可以参考[编译文档](/docs/install/source-install/compilation)，看看是否与您预想的一样工作。如果有问题，您可以直接联系我们，提 Issue 或者通过阅读和分析源代码自己修复。
```

**Comment by wangtianyi2004** [comment]
[typo](docs)Fix the jump link 404 in release-verify #13200

**Comment by wangtianyi2004** [comment]
[typo](docs)Fix the jump link 404 in release-verify #13200

**Pull request closed without merging**